src/GafferOSL/OSLObject.cpp

```diff
@@ -183,7 +190,15 @@ IECore::ConstObjectPtr OSLObject::computeProcessedObject( const ScenePath &path,
 
 	PrimitivePtr outputPrimitive = inputPrimitive->copy();
 
-	CompoundDataPtr shadedPoints = shadingEngine->shade( shadingPoints.get() );
+	ShadingEngine::Transforms transforms;
+
+	ShadingEngine::Transform world;
+	world.fromObjectSpace = inPlug()->fullTransform( path );
```

Comment:
Do you think we should add a constructor to Transform? Maybe a two-matrix version and a one-matrix version that automatically computes the inverse?